lib/internal/util.js

```diff
@@ -443,6 +455,7 @@ module.exports = {
   filterDuplicateStrings,
   getConstructorOf,
   getSystemErrorName,
+  getSystemErrorMap,
```

Comment:
nit: keep ASCII order
```suggestion
  getSystemErrorMap,
  getSystemErrorName,
```